src/app/common/hooks/account/use-account-names.ts

```diff
@@ -24,7 +25,7 @@ export function useAccountDisplayName({ address, index }: { index: number; addre
   const query = useGetBnsNamesOwnedByAddress(address, {
     select: resp => {
       const names = resp.names ?? [];
-      return names[0] || getAutogeneratedAccountDisplayName(index);
+      return formatAccountName(names[0]) || getAutogeneratedAccountDisplayName(index);
```

Comment:
`names[0]` can be `undefined` here, so `formatAccountName` will throw error

Reply:
Thank you for pointing that out, i will open a new PR with a validation added to the function

Reply:
@kyranjamie @Aman-zishan wonder if it's better to put up new pr to fix this?

Reply:
opened a new PR #5484

Reply:
no, we can just rebase this one

Reply:
yeah, it would be great. and could you please squash commits? I believe we can have just one commit for this change

Reply:
Alright , i will do that

Reply:
in the future we won't close your pr until you address reviews, this time was my fault, I just should have reviewed it earlier

Reply:
but yeah, if it even happens, feel free just to open new pr. so we can check and review again if necessary

Reply:
alright! no problems